content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -19,6 +19,12 @@ weight: 2
 
 This guide explains how to install RDI and integrate it with your source database.
 
+{{< note >}}RDI v1.2.7 had a CoreDNS issue where in some installations the host DNS could not
+resolve hostnames for the RDI components. This meant that it couldn't connect to the RDI database
+and resume the installation. This problem is fixed in v1.2.8, so we recommend you use this version
```

Comment:
I would lead this note with the recommendation to use the new version first, then explain why.

Reply:
Done.